Formula/gnuradio.rb

```diff
@@ -72,18 +65,11 @@ class Gnuradio < Formula
     sha256 "964031c0944f913933f55ad1610938105a6657a69d1ac5a6dd50e16a679104d5"
   end
 
-  # patch for boost 1.73.0, remove after next release
-  # https://github.com/gnuradio/gnuradio/pull/3566
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/0d2af1812716a874d1e49268e999ea1a8ca9fc3c/gnuradio/boost-1.73.0.patch"
-    sha256 "7e4abd08210d242d65807b7e2419f163a58b4630027a3beaff0e325d044266d7"
-  end
-
-  # Add -undefined dynamic_lookup linker flag back for macOS
-  # https://github.com/gnuradio/gnuradio/pull/3674
+  # patch to fix drag-and-drop in gnuradio-companion
+  # https://github.com/gnuradio/gnuradio/issues/2727
   patch do
-    url "https://github.com/gnuradio/gnuradio/commit/80ba62cb11cf604495e87a5e302e68eaf441eea9.patch?full_index=1"
-    sha256 "d12640f62b266b244950d84f2deb1544f41574229106a525e693159fb3fc80eb"
+    url "https://github.com/gnuradio/gnuradio/commit/5ee92ee7e2a9936615a494aa814fa31eda0398e7.patch?full_index=1"
+    sha256 "3d12bb3f7fdaf118fcff6993ff147f975b4aab13e5e7c76059d8866554135116"
```

Comment:
Since it's already merge we can use a commit from master
```suggestion
    url "https://github.com/gnuradio/gnuradio/commit/518dc7eda3a2575292dc67374cad62c887f83d12.patch?full_index=1"
    sha256 "88f28e204615c1893568cf72c3841cf372f7626e4e5aadd6a099fe4a3caa08a9"
```